scripts/konduit.sh

```diff
@@ -349,6 +426,7 @@ redis-cli)
 esac
 open_tunnels >/dev/null 2>&1
 sleep 5 # Need to allow the connections to open
+ps -ef |grep kubectl
```

Comment:
Debug code?

Reply:
yes, in the PR notes I have "remove extra debugging output" before merge